src/frontend/src/app/update-modal/update-modal.component.html

```diff
@@ -10,13 +10,13 @@
         <form [formGroup]="form" (ngSubmit)="f.form.valid && updateSchedule()" #f="ngForm" novalidate>
           <div class="columns">
             <div class="column">
-              <div class="field" [ngClass]="{ 'has-error': f.submitted && !form.controls.name.valid }">
```

Comment:
what is the impact of f.submitted ; we need to make sure; Cancel is not broken

Reply:
it was to keep track on whether form is submitted or not but f.submitted was unused here so I have removed that. Cancel in not broken and working fine.